RecoVertex/BeamSpotProducer/plugins/BeamSpotOnlineProducer.cc

```diff
@@ -1,4 +1,4 @@
-
+#include "DataFormats/L1GlobalTrigger/interface/L1GlobalTriggerEvmReadoutRecord.h"
```

Comment:
why is this needed here?
This include is present in RecoVertex/BeamSpotProducer/interface/BeamSpotOnlineProducer.h

Reply:
Because otherwise my checker would say that `DataFormats/L1GlobalTrigger` is not needed, because it considers `interface` to not be a part of the plugins. But the best solution is probably to merge the header file with the source, I'll do that.